src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -0,0 +1,252 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>7.3</LangVersion>
```

Comment:
Any reason to pin this here.  Can we let it float to arcade default?

Reply:
Ah we don't need this anymore. Yup, letting arcade control it now